include/silo/preprocessing/partition.h

```diff
@@ -21,71 +21,68 @@ class DatabaseConfig;
 
 namespace silo::preprocessing {
 
-class PangoLineageCounts;
-
-class Chunk {
+class LineageGroup {
    friend class Partition;
-   friend class boost::serialization::access;
-
-   template <class Archive>
-   [[maybe_unused]] void serialize(Archive& archive, [[maybe_unused]] const uint32_t version) {
-      // clang-format off
-      archive & prefix;
-      archive & count_of_sequences;
-      archive & offset;
-      archive & pango_lineages;
-      // clang-format on
-   }
 
    std::string prefix;
    uint32_t count_of_sequences;
-   uint32_t offset;
    std::vector<common::UnaliasedPangoLineage> pango_lineages;
 
-   Chunk() = default;
+   LineageGroup() = default;
 
   public:
-   Chunk(silo::common::UnaliasedPangoLineage lineage, uint32_t count);
-   Chunk(std::vector<silo::common::UnaliasedPangoLineage>&& lineages, uint32_t count);
+   LineageGroup(silo::common::UnaliasedPangoLineage lineage, uint32_t count);
+   LineageGroup(std::vector<silo::common::UnaliasedPangoLineage>&& lineages, uint32_t count);
 
-   void addChunk(Chunk&& other);
+   void addLineageGroup(LineageGroup&& other);
```

Comment:
This seems unused.

Reply:
Is `LineageGroup` used at all? I just deleted it and the code compiles without it.